migrations/20231211002759-create_users_attempt_change_email_table.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const tableName = 'users_attempt_change_email';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable(tableName, {
+      id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      user_id: {
```

Comment:
relate by uuid and establish the foreign key please

Reply:
I'm gonna do it, but, could you tell me why, please?

Reply:
An auto-incremental ID has a well-known limit where it overflows. Part of our legacy system involves using the users.id, files.id, etc, therefore, we need to stop creating new references while removing the already existent ones to use UUIDs, which use their data type that does not have a limit apart from using all the UUIDs (5.3 x 10^36 possibilities). 

If you reach the limit, you just cannot insert new records. We are far from that, but we should take care before it's near as it would mean we will have x100 more users and the risk would be imminent.